mods/gen3/items.js

```diff
@@ -216,6 +216,7 @@ exports.BattleItems = {
 		onEat: function (pokemon) {
 			this.heal(30);
 		},
+		desc: "Restores 30 of the holder's health when at 1/2 max HP or less. Single use.",
```

Comment:
I'd probably say `Restores 30 of the holder's HP when at 1/2 max HP or less. Single use.`